### PR TITLE
fix(argo-rollouts): Add subscriptions into notification-configmap

### PR DESCRIPTION
**Pull request by jaimeph**
Add subscriptions into notification-configmap.
https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/subscriptions/#default-subscriptions

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-rollouts/templates/controller/notifcations-configmap.yaml [hunk 1]
```diff
@@ -16,3 +16,7 @@ data:
   {{- with .Values.notifications.triggers }}
     {{- toYaml . | nindent 2 }}
   {{- end }}
+  {{- with .Values.notifications.subscriptions }}
+  subscriptions:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```
Comment (yu-croco) [inline charts/argo-rollouts/templates/controller/notifcations-configmap.yaml:22]: nits; please add new blank line in the end

File: charts/argo-rollouts/values.yaml [hunk 2]
```diff
@@ -477,3 +477,11 @@ notifications:
...
     #   - send: [my-purple-template]
     #     when: rollout.spec.template.spec.containers[0].image == 'argoproj/rollouts-demo:purple'
+
+  # -- The subscriptions define the subscriptions to the triggers in a general way for all rollouts
+  subscriptions: {}
+    # subscriptions:      
+    #   recipients:
+    #     - slack:<channel>
+    #   triggers:
+    #     - on-rollout-completed
```
Comment (yu-croco) [inline charts/argo-rollouts/values.yaml:487]: nits; please add new blank line in the end

Comment (jaimeph) [reply charts/argo-rollouts/values.yaml:487]: Done!

File: charts/argo-rollouts/templates/controller/notifcations-configmap.yaml (hunk 1 above)
Comment (jaimeph) [reply charts/argo-rollouts/templates/controller/notifcations-configmap.yaml]: Done!

File: charts/argo-cd/templates/argocd-server/certificate.yaml [hunk 3]
```diff
@@ -47,4 +47,4 @@ spec:
   usages:
     {{- toYaml . | nindent 4 }}
   {{- end }}
-{{- end }}
+{{- end }}
```
Comment (yu-croco) [inline charts/argo-cd/templates/argocd-server/certificate.yaml:50]: maybe wrong file diff?

File: charts/argo-rollouts/values.yaml (hunk 2 above)
Comment (yu-croco) [reply charts/argo-rollouts/values.yaml:487]: Are you sure...?
image

**Comment by jaimeph** [comment]
I close the pr and reopen one with a new branch with clean changes.

**Pull request closed without merging**